### PR TITLE
Add portable build to Windows

### PR DESCRIPTION
**Pull request by haslinghuis**
Follow up on: https://github.com/betaflight/betaflight-configurator/pull/2739

Fixes: #548

### CODE REVIEW

**Comment by TheIsotopes** [comment]
@haslinghuis for the portable version i would prefer a version with `nwBuilderOptions` `zip: false,`. Is that doable?

**Review by McGiverGim — APPROVED** [review]

**Comment by haslinghuis** [comment]
@TheIsotopes done. Thanks for the tip (as configurator is doing the same).

**Comment by McGiverGim** [comment]
I don't think we must use different parameters, we can't discuss to change it for all, but not only for portable.

**Comment by haslinghuis** [comment]
@McGiverGim configurator is using the same parameter. Otherwise happy to remove.

**Comment by TheIsotopes** [comment]
@McGiverGim as it is set now, it works on all versions. it should stay that way, it's the same as with the configurator.

**Comment by McGiverGim** [comment]
Long story short... if I remember correctly, in the Configurator was put `false` because for some user the startup time was very high (maybe some antivirus?). It has a drawback: the installer take a lot more because it registers any file that install.

https://github.com/betaflight/betaflight-configurator/pull/1057

The only problem is that the UPDATE of the app produced all the problems we suffered in the Configurator. I hope that with the new installer this does not happen anymore.

I prefer the compressed way, but if we want to use it uncompressed, is not a problem to me neither. But then we need to "uninstall" the Blackbox when installing to be sure to not let a lot of old files from different versions. Like we do in the Configurator. The installer only writes the new files, it does not remove the old one. When is compressed is not a problem, the files are the same, but now it will not.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
According to the linked issue `nwBuilderOptions zip: true` should work on both clients now???

**Comment by TheIsotopes** [comment]
> do you experience any issue with zip: true?

Actually no, but loading takes longer on older computers.
That's why i prefer `zip: false` only on portable version, fast loading and no long installation / deinstallation, can simply be deleted and that's it.

**Comment by haslinghuis** [comment]
@McGiverGim It's your take. I'm happy either way,

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by Asizon** [comment]
These changes are not aplying for mac users, should we aplied also for this SO?

File: gulpfile.js
Comment (Asizon) [inline gulpfile.js:36]: If we want to go with this, we need to do mcgiver requested changes on the installer side.

**Review by Asizon — CHANGES_REQUESTED** [review]

File: gulpfile.js
Comment (haslinghuis) [reply gulpfile.js]: Removed.

**Comment by McGiverGim** [comment]
My comment was, as Asizon said, because if we add the parameter, we need to modify the installer to uninstall the earlier before installing. A lot of garbage can remain if we don't do that. If we add that, no problem to use it uncompressed.

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
?

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Pull request merged**
